web/installer

```diff
@@ -52,10 +52,12 @@ function process($argv)
     }
 
     if ($ok || $force) {
-        installComposer($version, $installDir, $filename, $quiet, $disableTls, $cafile, $channel);
-        showWarnings($warnings);
-        showSecurityWarning($disableTls);
-        exit(0);
+        $installer = new Installer($quiet, $disableTls, $cafile);
+        if ($installer->run($version, $installDir, $filename, $channel)) {
```

Comment:
I know you didn't really change the flow... but it just seems weird to me that if `run()` succeeds (that is how I would interpret this), we then proceed to show warnings and such.. :P

I might take a stab at reflowing / refactoring some of this code after this gets merged.